packages/relay/src/index.ts

```diff
@@ -18,7 +18,7 @@
  *
  */
 
-import { Block, Log, Receipt, Transaction } from './lib/model';
+import { Block, Log, Receipt, Transaction } from './lib/models';
```

Comment:
I'd keep these models (i.e the JSON RPC response ones) and revert the mirror node ones for another PR per my other feedback

Reply:
Additionally, long term we'd want to consume the specs under https://github.com/ethereum/execution-apis/tree/main/src/schemas and generate the models this way and no longer needs these files.